core/tests/org.openjdk.jmc.flightrecorder.test/src/test/java/org/openjdk/jmc/flightrecorder/test/internal/util/ParserToolkitTest.java

```diff
@@ -0,0 +1,26 @@
+package org.openjdk.jmc.flightrecorder.test.internal.util;
```

Comment:
This file needs a copyright header and it needs to be moved to src/main from src/test as the project structure has been modified a little lately with https://github.com/openjdk/jmc/commit/6aeee1375d8ea5739ba2a85675fb5e00f1eea9af